### PR TITLE
Remove get_url from task locking

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #5148

### CODE REVIEW

**Review by lubosmj — COMMENTED** [review]
Are we going to change this part as well https://github.com/pulp/pulpcore/blob/778108af02e75a44d53a66749529e1b0acd751c7/pulpcore/app/replica.py#L59?

File: CHANGES/5148.removal [hunk 1]
```diff
@@ -0,0 +1 @@
+Removed ``get_url`` from task resource locking.
```
Comment (daviddavis) [inline CHANGES/5148.removal:1]: As a user, I have no idea what this means. Any chance you can spell out how the API will change (if it all)? Like it sounds like the `reserved_resources_record` field is changing on the `/pulp/api/v3/tasks/` is changing? Anything else?

Comment (gerrod3) [reply CHANGES/5148.removal]: @daviddavis I rewrote the changelog, can you check it out?

Comment (daviddavis) [reply CHANGES/5148.removal]: @gerrod3 looks great. Thanks!

**Review by mdellweg — APPROVED** [review]

**Review by lubosmj — APPROVED** [review]

**Pull request merged**